docs/concepts/security.md

```diff
@@ -1,6 +1,13 @@
 # Security
 
-## Content Security Policy
+See [https://github.com/ractivejs/ractive/issues/1897](https://github.com/ractivejs/ractive/issues/1897) for details.
+
+## Use of the Function constructor
+
+Ractive uses the `Function` constructor, an API similar to `eval`, to evaluate [template expressions](./templates.md#expressions_1). Because of this, it's easy to conclude that the library is insecure because of the popular saying "`eval` is evil". But that isn't always the case.
 
-To use Ractive with [Content Security Policy](http://www.html5rocks.com/en/tutorials/security/content-security-policy/), you'll currently need `'unsafe-eval'` specified for `scriptSrc` in your CSP header. This may change in future - see https://github.com/ractivejs/ractive/issues/1897 .
+`eval` and friends can lead to security issues when it's allowed to execute arbitrary strings as scripts in the context of the current user. However, the key word here is _arbitrary_. The risk is definitely greater when code and/or data come from untrusted and/or unsanitized sources. This can be mitigated employing best practices such as code audits and security testing, and making sure the app only interfaces with trusted sources. While this does not eliminate the possibility of XSS while using the library, at least it reduces the risk of one happening.
+
+## Content Security Policy
 
+Because of Ractive's use of `eval`-like APIs, using Ractive under [Content Security Policy](http://www.html5rocks.com/en/tutorials/security/content-security-policy/) will require [`'unsafe-eval'`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#Unsafe_eval_expressions) specified for [`script-src`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src). This may change in future.
```

Comment:
That's probably not going to change in the future because the parser defaults to `csp` and pre-parsing templates for production code is a good idea for performance if nothing else. We should probably add a "deploying ractive apps in production" somewhere.

Also, if you deploy the runtime-only flavor of ractive, I think _all_ of the parsing bits are left off, including computation shorthad, so there is no use of eval anywhere.